app/src/main/java/com/openclassrooms/realestatemanager/ui/EstateListAdapter.kt

```diff
@@ -53,4 +71,16 @@ class EstateListAdapter(
             Estate.Type.PENTHOUSE -> resources.getString(R.string.estate_type_penthouse)
         }
     }
+
+    fun updateSelectedEstateColor(selectedEstate: Estate?) {
+        this.lastSelectedEstate = this.selectedEstate
+        this.selectedEstate = selectedEstate
+        if (lastSelectedEstate != null) {
+            notifyItemChanged(currentList.indexOf(lastSelectedEstate))
+        }
+        if (selectedEstate != null) {
+            notifyItemChanged(currentList.indexOf(selectedEstate))
+        }
```

Comment:
```suggestion
        val lastSelected = this.selectedEstate
        this.selectedEstate = selectedEstate
        if (lastSelectedEstate != null) {
            notifyItemChanged(currentList.indexOf(lastSelectedEstate))
        }
        if (selectedEstate != null) {
            notifyItemChanged(currentList.indexOf(selectedEstate))
        }
```

Si tu veux éviter de garder last en mémoire

Reply:
Je l'utilise aussi dans le onBindViewHolder